### ISSUE TITLE
Add HOSTNAME to the webpage header ?

### ISSUE THREAD

**Issue by KalixtShawxo** [ravenclaw900/DietPi-Dashboard#75]
Just a small enhancement request. I am running DietPi on several SBCs, and I love using this dashboard, and sometimes have multiple open.  But in the browser tabs they all just say "DietPi Dashboard" (with the IP if you mouse over) but it would be nice if in the webpage itself in the header to the right of the DietPi icon it could display the hostname of the SBC it's connected to.
I find myself often jumping to the "management" section to verify which SBC i am on at the moment.

Thanks for any consideration.

**Comment by ravenclaw900** [comment]
Hi,
The "multiple nodes" feature (https://github.com/ravenclaw900/DietPi-Dashboard/pull/66) looks useful for you. I think it would be best to show the IP (probably optionally) if you were using this feature with multiple nodes, what do you think?

**Comment by KalixtShawxo** [comment]
Multiple nodes from one interface would be great, and definitely useful for me having 1/2 dozen DietPi SBCs running. The hostname (and optionally the IP) being prominent on whatever screen you are on would definitely be needed to make sure you don't make a change on the wrong system.  I'll definitely take a look at #66.

**Comment by Joulinar** [comment]
just to give this a push. Especially with the new feature to show more nodes in one Gui. It would be helpful to have a identifier next to the IP address on the node section menu. It could be the hoste name of the node. But I guess this would require to ask als nodes during service start up. Another option would be to allow users to specify a name inside node configuration like

```
nodes = ["dietpi3.lan:5252,node1", "192.168.x.x:4386,node2"]
```

As well it might be a good idea to allow to specify a name for the main frontend server. Actually it is displayed with IP address only on the node selection menu 😎

**Comment by KalixtShawxo** [comment]
Hello!  Finally got back to messing around with things.... was sick for a bit and some other issues. 

Anyway, I like the added Address at the top bar for selecting, but when I select something other than the default, the address selection box disappears and I have to reload the original page. 

Also, I then tried doing the nodes with a name in the config file and that totally breaks it.  the selection box doesn't work at all.

Here's what works to show the different graphs, (on 192.168.2.11) but after selecting the box disappears:
`nodes = ["192.168.2.13:5252", "192.168.2.14:5252"]`

then trying this just breaks it completely:
`nodes = ["192.168.2.13:5252,TARDIS", "192.168.2.14:5252,Tachikoma"]`

**Comment by ravenclaw900** [comment]
Hmm, the box shouldn't disappear. Are either of those frontend nodes? Those currently aren't supported for viewing from other pages. The named syntax hasn't been implemented yet, so it won't work at all.

**Comment by KalixtShawxo** [comment]
Nope, both were reinstalled and set as no frontend prior to adding them to the config for the third one. 
The box disappears after selection, but it does work to bring up the graphs for the other dashboard. Which is verified by going into the management screen. 
I misread your previous post and thought the naming was an option you had put in, not an idea for future.
I will recheck it all tonight and see what it does. 

Sent from Yahoo Mail on Android 
 
  On Wed, Feb 23, 2022 at 7:23, ***@***.***> wrote:   


Hmm, the box shouldn't disappear. Are either of those frontend nodes? Those currently aren't supported for viewing from other pages. The named syntax hasn't been implemented yet, so it won't work at all.

**Comment by KalixtShawxo** [comment]
well after a reboot or 3....including several power outages yesterday that corrupted a few of my SBCs.... it is working as expected, except for the remote that was corrupted. it no longer deletes the selection box.

**Comment by ravenclaw900** [comment]
That's odd... Did you have to reinstall the dashboard on any of them?

**Comment by KalixtShawxo** [comment]
not since the reported issue.....but I did do reinstalls prior setting up the one to read the others. 
I did the reinstalls on the 2 and made them back end only.   then after that I setup a third one to read the other two. 
That is where i was with the reported issue above.  today, I was checking my SBC cause I had storm power outages and found a couple were corrupted, including one of the two I setup for the remote dashboard.  but the funny thing is now the other that works, works fine as expected with the selection dropbox staying in place. 
Now since I had responded above, I have setup another Pi to be remotely dashed (for lack of a better term). 
I set it up by doing a reinstall to remove the front end, then I went into my remote dashboard system I had setup before and added the new IP to the config.  rebooted and same thing, it works, but the selection dropbox disappears after selection but only for the one new one. the ones that were working still do. 

so I'm at a loss again.  I'm working on another issue rebuilding a Plex server and will come back to this issue over the weekend to see if it "fixes itself" like the others did.
